integrations/salesforce/grpc.go

```diff
@@ -0,0 +1,45 @@
+package salesforce
+
+import (
+	"context"
+	"crypto/tls"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+type salesforceAuth struct {
+	accessToken string
+	instanceURL string
+	tenantID    string
+}
+
+func initConn(accessToken, instanceURL, orgID string, l *zap.Logger) (*grpc.ClientConn, error) {
+	conn, err := grpc.NewClient(
+		"api.pubsub.salesforce.com:443",
+		grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})),
+		grpc.WithPerRPCCredentials(&salesforceAuth{
+			accessToken: accessToken,
+			instanceURL: instanceURL,
+			tenantID:    orgID,
+		}),
+	)
+	if err != nil {
+		l.Error("failed to create gRPC connection", zap.Error(err))
```

Comment:
If we see this in Datadog, it would to be a little more specific
```suggestion
		l.Error("failed to create gRPC connection for Salesforce events", zap.Error(err))
```

Reply:
Done